### PR TITLE
FIX: Window sorting

### PR DESCRIPTION
**Pull request by mizaki**
I removed the sorting when creating the update on click. This puts it back.

### CODE REVIEW

File: comictaggerlib/seriesselectionwindow.py [hunk 1]
```diff
@@ -534,18 +534,20 @@ def update_row(self, row: int, series: ComicSeries) -> None:
...
-        item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
-        item.setData(QtCore.Qt.ItemDataRole.DisplayRole, series.count_of_issues)
-        item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
+        if series.count_of_issues is not None:
+            item_text = f"{series.count_of_issues:04}"
+            item = self.twList.item(row, 2)
+            item.setText(item_text)
+            item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
+            item.setData(QtCore.Qt.ItemDataRole.DisplayRole, series.count_of_issues)
+            item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
```
Comment (lordwelch) [inline comictaggerlib/seriesselectionwindow.py]: These should always update, otherwise we can end up with old data. I don't believe it will with the current code but we should do it properly.

File: comictaggerlib/issueselectionwindow.py [hunk 2]
```diff
@@ -161,7 +161,10 @@ def perform_query(self) -> None:
         for row, issue in enumerate(self.issue_list.values()):
             self.twList.insertRow(row)
             for i in range(3):
-                self.twList.setItem(row, i, QtWidgets.QTableWidgetItem())
+                if i == 0:
+                    self.twList.setItem(row, i, IssueNumberTableWidgetItem())
+                else:
+                    self.twList.setItem(row, i, QtWidgets.QTableWidgetItem())
```
Comment (lordwelch) [inline comictaggerlib/issueselectionwindow.py]: The loop should be unwound it even makes for shorter code in this case.

File: comictaggerlib/seriesselectionwindow.py [hunk 3]
```diff
@@ -534,18 +534,20 @@ def update_row(self, row: int, series: ComicSeries) -> None:
...
-        item_text = str(series.start_year)
-        item = self.twList.item(row, 1)
-        item.setText(item_text)
-        item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
-        item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
+        if series.start_year is not None:
+            item_text = f"{series.start_year:04}"
+            item = self.twList.item(row, 1)
+            item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
+            item.setData(QtCore.Qt.ItemDataRole.DisplayRole, series.start_year)
```
Comment (lordwelch) [inline comictaggerlib/seriesselectionwindow.py]: Why the change from setText to setData

**Review by lordwelch — CHANGES_REQUESTED** [review]

File: comictaggerlib/seriesselectionwindow.py (hunk 3 above)
Comment (mizaki) [reply comictaggerlib/seriesselectionwindow.py]: I imagine it was where I was trying to break apart the display text from the sorting. Reverted.
